docs/cli/geo/search.md

```diff
@@ -0,0 +1,146 @@
+❯ No, I only need to track consumers personal mobile devices 
+  Yes, I track commercial assets (For example, any mobile object that is tracked by a company in support of its business)
+
+Successfully set RequestBasedUsage pricing plan for your Geo resources.
+```
+
+```console
+? Are you tracking commercial assets for your business in your app?
+  No, I only need to track consumers personal mobile devices 
+❯ Yes, I track commercial assets (For example, any mobile object that is tracked by a company in support 
+  of its business)
+
+? Does your app provide routing or route optimization for commercial assets? (y/N) no
+
+Successfully set MobileAssetTracking pricing plan for your Geo resources.
+```
+
+```console
+? Are you tracking commercial assets for your business in your app?
+  No, I only need to track consumers personal mobile devices 
+❯ Yes, I track commercial assets (For example, any mobile object that is tracked by a company in support 
+  of its business)
+
+? Does your app provide routing or route optimization for commercial assets? (y/N) yes
+
+Successfully set MobileAssetManagement pricing plan for your Geo resources.
+```
+
+We advice you to go through the [location service pricing](https://aws.amazon.com/location/pricing/) along with the [location service terms](https://aws.amazon.com/service-terms/) (_82.5 section_) to learn more about the pricing plan.
+
+### Advanced Settings
+You can optionally set additional properties of the location search index. If you choose to skip, defaults will be set for these properties. 
+
+```console
+Available advanced settings:
+- Search data provider (default: Esri)
+- Search result storage location (default: no result storage)
+
+? Do you want to configure advanced settings? 
+> Yes
+```
+
+#### Search data provider
+You can select a data provider as the source for geocoding, reverse geocoding and searches.
+Each provider gathers and curates their data using different means. They may also have varying expertise in different regions of the world.
+The available data providers of GeoSpatial data are shown. To learn more about data providers, please refer this [location service doc](https://docs.aws.amazon.com/location/latest/developerguide/what-is-data-provider.html).
+
+```console
+? Specify the data provider of GeoSpatial data for this Search Index:
+❯ Esri
+  Here
+```
+
+`Esri` will be set as default data provider for the location search index, if you do not want to explicitly set this property.
+
+#### Search result storage location
+You can specify how the results of a search operation will be stored by the caller.
+```console
+? Specify the data storage option for requesting Places:
+> SingleUse
+  Storage
+```
+`SingleUse` specifies that the results won't be stored. This will be chosen as default if the developer does not want to explicitly set this property. 
+`Storage` specifies that the result can be cached or stored in a database.
```

Comment:
in a database. Which one?

Reply:
No longer needed as the walkthrough question is modified. Provided a link to refer for additional information.